lib/MusicBrainz/Server/Form/User/Report.pm

```diff
@@ -26,16 +26,16 @@ has_field 'reveal_address' => (
 );
 
 Readonly our %REASONS => (
-    'spam' => l('Editor is spamming'),
-    'unresponsiveness' => l('Editor is unresponsive to edit notes'),
-    'ignoring_guidelines' => l('Editor intentionally ignores accepted guidelines'),
-    'enforcing_guidelines' => l('Editor is overzealous in enforcing guidelines as rules'),
-    'voting' => l('Editor engages in overzealous or abusive yes/no voting'),
-    'other' => l('Editor has violated some other part of our Code of Conduct'),
```

Comment:
These should technically be changed to `N_l` instead (they're only extracted incidentally because the ones in root/user/ReportUser.js already have `N_l`).